src/modules/CaaSMapper.ts

```diff
@@ -372,7 +380,20 @@ export class CaaSMapper {
       previewId: this.buildPreviewId(item.identifier),
       meta: await this.mapDataEntries(item.metaFormData, [...path, 'meta']),
       description: item.description,
-      resolutions: item.resolutionsMetaData,
+      resolutions: Object.entries(item.resolutionsMetaData)
```

Comment:
This whole function is quite short for what it does which is nice, but it is also very hard to read – especially if you're not familiar with the code in general. I would suggest coding this functionality a bit more readable.

Reply:
This has been refactored.